[32m2025-04-14 05:25:57 +0000[0m - dagster - [34mDEBUG[0m - internal_preprocessing_job - 80b28534-3931-4da2-be1c-d06a7a6396e0 - 31326 - LOGS_CAPTURED - Started capturing logs in process (pid: 31326).
[32m2025-04-14 05:25:57 +0000[0m - dagster - [34mDEBUG[0m - internal_preprocessing_job - 80b28534-3931-4da2-be1c-d06a7a6396e0 - 31326 - internal_raw_sales_data - STEP_START - Started execution of step "internal_raw_sales_data".
[32m2025-04-14 05:25:57 +0000[0m - dagster - [34mINFO[0m - internal_preprocessing_job - 80b28534-3931-4da2-be1c-d06a7a6396e0 - internal_raw_sales_data - Reading need state sales data
[32m2025-04-14 05:25:57 +0000[0m - dagster - [34mDEBUG[0m - internal_preprocessing_job - 80b28534-3931-4da2-be1c-d06a7a6396e0 - 31326 - internal_raw_sales_data - STEP_OUTPUT - Yielded output "result" of type "SalesSchema". Warning! Type check failed.
[32m2025-04-14 05:25:57 +0000[0m - dagster - [34mERROR[0m - [31minternal_preprocessing_job - 80b28534-3931-4da2-be1c-d06a7a6396e0 - 31326 - internal_raw_sales_data - STEP_FAILURE - Execution of step "internal_raw_sales_data" failed.

dagster._core.errors.DagsterTypeCheckDidNotPass: Type check failed for step output "result" - expected type "SalesSchema". Description: Unexpected error during validation: SKU_NBR

Resolved plan until failure:

	---> FAILED HERE RESOLVING 'sink' <---
DF ["STORE_NBR", "PRODUCT_ID", "SALES", "DATE"]; PROJECT */4 COLUMNS

Stack Trace:
  File "/workspaces/testing-dagster/.venv/lib/python3.10/site-packages/dagster/_core/execution/plan/execute_plan.py", line 245, in dagster_event_sequence_for_step
    yield from check.generator(step_events)
  File "/workspaces/testing-dagster/.venv/lib/python3.10/site-packages/dagster/_core/execution/plan/execute_step.py", line 508, in core_dagster_event_sequence_for_step
    for evt in _type_check_and_store_output(step_context, user_event):
  File "/workspaces/testing-dagster/.venv/lib/python3.10/site-packages/dagster/_core/execution/plan/execute_step.py", line 552, in _type_check_and_store_output
    yield from _type_check_output(step_context, step_output_handle, output)
  File "/workspaces/testing-dagster/.venv/lib/python3.10/site-packages/dagster/_core/execution/plan/execute_step.py", line 425, in _type_check_output
    raise DagsterTypeCheckDidNotPass(
[0m
